chapter2-segwitv0/p2wsh-2-of-2-multisig.ipynb

```diff
@@ -477,7 +477,7 @@
     "    + bytes.fromhex(\"00\") # Add an extra \"00\" for the CheckMultiSig bug\n",
     "    + pushbytes(signature1)\n",
     "    + pushbytes(signature2)\n",
-    "    + pushbytes(redeemScript)\n",
+    "    + varint_len(redeemScript) + redeemScript\n",
```

Comment:
```suggestion
    "    + varint_len(redeemScript)\n",
    "    + redeemScript\n",
```

Reply:
nit: could you put them all on the same column just to be consistent with the rest of the notebook.